CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Changelog
 
 ## In Development
-
+* Add ability to specify service type for st2web (#35)
```

Comment:
Probably need to add a new version `v0.8.0` here too, as it was bumped in Helm metadata already